### PR TITLE
Branch view command

### PR DESCRIPTION
**Pull request by wassilililily**
Implements View Company Command feature to see all the jobs and contacts associated with the company.

Currently there is a bug where user need to keep open up all the lists to use the view company command again properly.

### CODE REVIEW

**Review by Quasant — APPROVED** [review]
Okay, looks good from just inspecting the code.

File: src/main/java/seedu/address/logic/parser/ViewCompanyCommandParser.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+/**
+ * Parses input arguments and creates a new ViewCompanyCommand object.
+ */
+public class ViewCompanyCommandParser implements Parser<ViewCompanyCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of a ViewCompanyCommand
+     * and returns a ViewCompanyCommand object for execution.
+     *
+     * @throws ParseException if the user input does not conform to the expected format
```
Comment (AlekKwek18) [inline src/main/java/seedu/address/logic/parser/ViewCompanyCommandParser.java:20]: Capital is needed for the first character after the variable/exception. Add period after the sentence ends.

i.e
@throws ParseException If the user input does not conform to the expected format.

**Review by AlekKwek18 — APPROVED** [review]
Overall LGTM, fix the comment issue and merge conflicts then merge and close the PR.

File: src/main/java/seedu/address/logic/commands/ViewCompanyCommand.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+ * Displays all jobs and persons linked to a specific company in the address book.
+ * <p>
+ * The {@code ViewCompanyCommand} identifies the company by its index in the currently displayed
+ * company list. It then filters and displays the jobs and persons associated with the selected company.
+ */
+public class ViewCompanyCommand extends Command {
+
+    public static final String COMMAND_WORD = "view";
+
+    public static final String ENTITY_WORD = "company";
```
[the remark this answers is not in the document]
Comment (KengHian) [reply src/main/java/seedu/address/logic/commands/ViewCompanyCommand.java:24]: What would you suggest the full command to be? just view <COMPANY_INDEX> ?

[the remark this answers is not in the document]
Comment (Quasant) [reply src/main/java/seedu/address/logic/commands/ViewCompanyCommand.java:24]: Either method works, so its not too big a deal. It's mostly about that one keystroke in "view 5" vs "view company 5" for instance.

**Pull request merged**